automerge/Cargo.toml

```diff
@@ -34,6 +34,7 @@ tracing-subscriber = {version = "0.2", features = ["chrono", "env-filter", "fmt"
 unicode-segmentation = "1.7.1"
 maplit = "^1.0.2"
 smol_str = "0.1.17"
+anyhow = "1.0.41"
```

Comment:
This should be an example only dependency.

Reply:
This is already in the `dev-dependencies` section, I'm not aware of an examples only one?

Reply:
Ah, right you are, I assumed you could have dependencies scoped by target. `dev-dependencies` seems fine.